README.md

```diff
@@ -391,6 +391,8 @@ installed as the `jaxlib` package. Use the following instructions to install a
 binary package with `pip` or `conda`, or to [build JAX from
```

Comment:
Work the sentence below in here:

Use the following instructions to install a binary package with `pip` or `conda`, to use a `Docker` container, or to ...